edk2toollib/windows/policy/firmware_policy_tool.py

```diff
@@ -0,0 +1,73 @@
+# @file
```

Comment:
i don't think this tool belongs here.  CLI / tools go into pytool extensions

Reply:
Removed in 2a966083e3b43c04a5f6c6ab338108bcdfe9244c